README.md

```diff
@@ -120,6 +120,32 @@ make install
 
 Please see our [additional configuration instructions](http://docs.timescale.com/getting-started/installation#update-postgresql-conf).
 
+#### Building from source (Windows)
+
+**Prerequisites**:
+
+- A standard [PostgreSQL 9.6 64-bit installation](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads#windows) 
+- Microsoft Visual Studio
+- [CMake](https://cmake.org/) version 3.4 or greater
+- Make sure all relevant binaries are in your PATH: `pg_config`, `cmake`, `MSBuild`
+
+```bash
+git clone git@github.com:timescale/timescaledb.git
+cd timescaledb
+
+# Bootstrap the build system
+./bootstrap.bat
```

Comment:
I built this in VS2017 by simply checking out and opening the project in the GUI and building. The CMake integration took care of the rest. Should we add this as an option?